src/scripts/emailBreachAlerts.js

```diff
@@ -311,18 +311,18 @@ if (process.env.NODE_ENV !== "test") {
           "env vars not set: GCP_PUBSUB_PROJECT_ID and GCP_PUBSUB_SUBSCRIPTION_NAME",
         );
       }
-      Sentry.captureCheckIn({
-        checkInId,
-        monitorSlug: SENTRY_SLUG,
-        status: "ok",
-      });
     })
     .catch((err) => console.error(err))
     .finally(async () => {
       // Tear down knex connection pools
       await knexSubscribers.destroy();
       await knexEmailAddresses.destroy();
       await knexHibp.destroy();
+      Sentry.captureCheckIn({
+        checkInId,
+        monitorSlug: SENTRY_SLUG,
+        status: "ok",
```

Comment:
Q: Do we know if the `status` is "ok" (if we hit the `catch` block)?

Reply:
`finally` block should run regardless of whether the `try` or `catch` blocks ran:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#finallystatements